### PR TITLE
VB-61: add avatar component

### PR DESCRIPTION
**Pull request by dannleed**

### CODE REVIEW

File: packages/frontend/components/header/components/avatar/styles.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { css } from '@emotion/react';
+import type { Theme } from 'theme';
+
+export const avatar = ({ spaces, radiuses }: Theme) => css`
+  display: flex;
+  align-items: center;
```
Comment (Nightlngale) [inline packages/frontend/components/header/components/avatar/styles.ts:6]: add shrink: 0

File: packages/frontend/components/header/components/avatar/component.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+}: AvatarProps) => (
+  <div css={styles.wrapper} {...props}>
+    <Image
+      css={styles.avatar}
+      src={image}
+      alt={`${firstName} ${lastName}`}
+      height={35}
+      width={35}
+    />
+    <span css={styles.name}>{`${firstName} ${lastName}`}</span>
```
Comment (Nightlngale) [inline packages/frontend/components/header/components/avatar/component.tsx:19]: use only first letters like in telegram

File: packages/frontend/components/header/components/avatar/component.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import Image from 'next/image';
+import * as styles from './styles';
+import type { AvatarProps } from './types';
+
+export const Avatar = ({
+  firstName,
+  lastName,
+  image,
+  ...props
```
Comment (Nightlngale) [inline packages/frontend/components/header/components/avatar/component.tsx:9]: we don't need ...props here

File: packages/frontend/components/header/components/avatar/component.tsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+}: AvatarProps) => (
+  <div css={styles.wrapper} {...props}>
+    <Image
+      css={styles.avatar}
+      src={image}
+      alt={`${firstName} ${lastName}`}
```
Comment (Nightlngale) [inline packages/frontend/components/header/components/avatar/component.tsx:15]: alt - avatar image

**Review by Vitalii4as — APPROVED** [review]

**Comment by dannleed** [comment]
image

**Review by Nightlngale — APPROVED** [review]

**Pull request merged**